src/apis/Api.ts

```diff
@@ -0,0 +1,9 @@
+import HTTPClient from "./HTTPClient";
+
+class Api extends HTTPClient {
+  public example = async () => "Example call";
+}
+
+const api = new Api(process.env.VITE_API_URL as string, true);
```

Comment:
a null check for this API url would be good, instead of casting to string

Reply:
👍